frontend/src/components/shared/select/Select.tsx

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './select.scss';
 
 type SelectProps = {
   name: string;
+  options: string[];
+  label?: string;
+  value?: string;
+  onChange?: (e: any) => void;
 };
+
+const getOptions = (arr: (string | number)[]) => {
+  return arr.map((item, i) => (
+    <option key={`${item}${i}`} value={item}>
+      {item}
+    </option>
+  ));
+};
```

Comment:
why do we need separate function? It is used only in one place
<select className="select" onChange={props.onChange} name={props.name} value={props.value}>
        {props.options.map((item, i) => (
    <option key={`${item}${i}`} value={item}>
      {item}
    </option>}
      </select>